zebra-consensus/src/script.rs

```diff
@@ -20,12 +30,14 @@ use crate::BoxError;
 /// [RFC4]: https://zebra.zfnd.org/dev/rfcs/0004-asynchronous-script-verification.html
 #[derive(Debug, Clone)]
 pub struct Verifier<ZS> {
-    state: ZS,
+    state: Timeout<ZS>,
 }
 
 impl<ZS> Verifier<ZS> {
     pub fn new(state: ZS) -> Self {
-        Self { state }
+        Self {
+            state: Timeout::new(state, UTXO_LOOKUP_TIMEOUT),
```

Comment:
Note: Due to the mapping in the BlockVerifier, timeout errors should appear as `VerifyBlockError::Transaction(Elapsed)`.

Ideally, we'd say that the UTXO request elapsed, but that doesn't really matter until we have a mempool. I've opened #1401 in the first stable release for follow-up.